Netezos.Tests/Rpc/settings.json

```diff
@@ -1,4 +1,11 @@
 {
+  "sandboxNode" : {
```

Comment:
It would be nice to be able to switch between different environments (sandbox/public node) using the config, e.g.:

instead of
  "sandboxNode" : {
    "imageName": "bakingbad/sandboxed-node",
    "tag" : "v9.1-1",
    "port" : "8732",
    "host" : "http://localhost",
    "healthCheckOnStartedTimeout": 20
  },
  "node": "https://mainnet-tezos.giganode.io/",

do
"node": {
  "type": "sandbox",
  "config": {
    "imageName": "bakingbad/sandboxed-node",
    "tag" : "v9.1-1",
    "port" : "8732",
    "host" : "http://localhost",
    "healthCheckOnStartedTimeout": 20,
    ...
  }
}

or

"node": {
  "type": "???",
  "config": {
    "url": "https://mainnet-tezos.giganode.io",
    "timeout" : 60,
    ...
  }
}